contrib/test_jenkins.sh

```diff
@@ -1161,11 +1161,26 @@ then
     set_ucx_common_test_env
 
     if [[ "$PROTO_ENABLE" == "no" ]]; then
-        run_test_proto_disable
+        run_test_proto_disable || {
```

Comment:
Because if you want to debug this kind of scenario (happened to me recently), you need to go to the `test_jenkins.sh` script and find the exact flag (for example: "ASAN_CHECK=yes") to reproduce it with. 
By adding this error log, you can just copy-paste and reproduce it.